plugins/twitter.py

```diff
@@ -56,7 +56,7 @@ def __init__(self,
         self.access_token = twitter_access_token
         self.access_token_secret = twitter_access_token_secret
         self.regex = re.compile(
-            r'https?://(mobile.)twitter.com/(?P<user>\w+?)/status/(?P<id>\d+)/?')
+            r'https?://(mobile\.)twitter.com/(?P<user>\w+?)/status/(?P<id>\d+)/?')
```

Comment:
Still needs a `?` after `(mobile\.)`, otherwise it will reject non-mobile links.